packages/plugin-vue-jsx/package.json

```diff
@@ -28,12 +28,15 @@
   },
   "dependencies": {
     "@rsbuild/plugin-babel": "workspace:*",
-    "@vue/babel-plugin-jsx": "1.2.1"
+    "@vue/babel-plugin-jsx": "1.2.1",
+    "babel-plugin-vue-jsx-hmr": "1.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.23.2",
     "@rsbuild/core": "workspace:*",
+    "@rspack/core": "0.5.5",
```

Comment:
This seems like an unnecessary dependency

Reply:
Indeed, I should have added it when I was testing, sorry about that

Reply:
Haha never mind